docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.    number_samples_target = [8] * 5
</pre><p>can you use a different number of samples to clearly show that it is not necessary to have the same shape for source and target?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>